Subject: [PATCH] powerpc: hide CONFIG_ALTIVEC on iSeries                                                                                                             
From: olh@suse.de

Do not offer CONFIG_ALTIVEC for legacy iSeries. They predate the 970.

Signed-off-by: Olaf Hering <olh@suse.de>

 arch/powerpc/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.15-rc4-olh/arch/powerpc/Kconfig
===================================================================
--- linux-2.6.15-rc4-olh.orig/arch/powerpc/Kconfig
+++ linux-2.6.15-rc4-olh/arch/powerpc/Kconfig
@@ -169,7 +169,7 @@ config PHYS_64BIT
 
 config ALTIVEC
 	bool "AltiVec Support"
-	depends on 6xx || POWER4
+	depends on (6xx || POWER4) && !PPC_ISERIES
 	---help---
 	  This option enables kernel support for the Altivec extensions to the
 	  PowerPC processor. The kernel currently supports saving and restoring
